packages/graphql/src/nexus.ts

```diff
@@ -1979,6 +1979,19 @@ export const Mutation = objectType({
         return ParticipantGroupService.submitGroupActivityDecisions(args, ctx)
       },
     })
+
+    t.field('changeCourseColor', {
+      type: Course,
+      args: {
+        courseId: nonNull(idArg()),
+        color: nonNull(
```

Comment:
Run `npm run format` and commit separately, I think prettier would put this on the same line